api/experience/repository_postgres.go

```diff
@@ -151,3 +154,23 @@ func (repo *experienceRepositoryPostgres) GetAllUserExperience(userId uint, limi
 
 	return exp, uint(total), err
 }
+
+func (repo *experienceRepositoryPostgres) AddSkillsToExperience(expID uint, skillIDs []uint) error {
+	var experience Experience
+	if err := repo.db.Where("id = ? ", expID).Find(&experience).Error; err != nil {
+		return fmt.Errorf("experience not found: %v", err)
+	}
+
+	for _, skillID := range skillIDs {
```

Comment:
we need to handle it in db transaction method
if any entry fails to it should not be commited